charts/prometheus/templates/server/cm.yaml

```
@@ -5,6 +5,9 @@ kind: ConfigMap
 metadata:
   labels:
     {{- include "prometheus.server.labels" . | nindent 4 }}
+{{- if .Values.server.extraConfigmapLabels }}
+{{ toYaml .Values.server.extraConfigmapLabels | indent 4 }}
+{{- end }}
```

Better with `with`. This code below will eliminate extra `{}` while calling `helm template <path to prometheus chart>`:

```suggestion
    {{- with .Values.server.extraConfigmapLabels }}
    {{- toYaml .Values.server.extraConfigmapLabels | nindent 4 }}
    {{- end }}
```

---

Hi, thank you for suggestion. 
I amended the commit, but the syntax there is different as `with` changes the scope.

```yaml
    {{- with .Values.server.extraConfigmapLabels }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
```